src/theme.js

```diff
@@ -1,10 +1,54 @@
 import { Link } from "gatsby"
 import React from "react"
-import styled from 'styled-components'
+import styled, { createGlobalStyle } from 'styled-components'
 
 const mediumSize = "770px"
 const smallSize = "450px"
 
+const lightScheme = `
+  body {
+    --bg: #ffffff;
+    --red: #ac3232;
+    --blue: #5656ff;
+    --sky-blue: #a8dbff;
+    --white: #ffffff;
+    --black: #000000;
+    --dark: #222222;
+    --grey: #e4e4e4;
+    --soft-grey: #f5f5f5;
+    background-color: var(--bg);
```

Comment:
red -> accent
blue -> hyperlink
sky-blue -> toggle ??
white -> text-body??
...

Reply:
name colors by functionality, not by color's own name